{
  try {
    underTest.selectByLogin(session,"unknown");
    fail();
  }
 catch (  Exception e) {
    assertThat(e).isInstanceOf(RowNotFoundException.class).hasMessage("User with login 'unknown' has not been found");
  }
}
